terminologies/technique/highResoluted3DReconstruction.jsonld

```diff
@@ -0,0 +1,8 @@
+{
+ "@id":"https://openminds.ebrains.eu/controlledTerms/technique/highResoluted3DReconstruction",

```

Comment:
I would delete this one and use "digitalVolumeReconstruction" from above instead.

@tgbugs: what do you think?